src/Certificate.php

```diff
@@ -81,7 +81,14 @@ public function getParentCertificateUrl(): string
 
     public function fetchParentCertificate(): self
     {
-        return static::loadFromUrl($this->getParentCertificateUrl());
+        $url = $this->getParentCertificateUrl();
+
+        $scheme = parse_url($url, PHP_URL_SCHEME);
+        if ($scheme !== 'http' && $scheme !== 'https') {
```

Comment:
Could you refactor this to an `in_array` check?

Reply:
Sure! I've changed the mentioned code and added an explaining comment.